apis/python/node/src/lib.rs

```diff
@@ -24,9 +25,10 @@ use pyo3::types::{PyBytes, PyDict};
 /// ```
 ///
 #[pyclass]
+#[derive(DirHelper)]
 pub struct Node {
     events: Events,
-    node: DoraNode,
+    pub node: DoraNode,
```

Comment:
Only document `Node::node`. The implementation in `pyo3_special_method_derive` calls the `Debug` impl of `DoraNode`. This will still look like Rust to Python users, which is something we want to avoid. Perhaps we can implement a recursive method to `pyo3_special_method_derive`?

Reply:
What you're trying to say is that because DoraNode is a rust object it's going to look not great, right?

Can we skip both and have something in the likes of:

```python
>>> node = Node()
>>> node
Node()
```

With just empty parenthesis?

In the future, I can add additional fields that I might want to expose to the user?

Reply:
Yes, `pyo3_special_method_derive` automatically skips fields which are not `pub`. If you add additional fields in the future,  you can add additional fields and as long as they are `pub` and use the derives it will work.